MaritzSTL/koa-ms
# Example

To test:
* `yarn run start:all` (this might need some timing rework, basically get it going)
* `yarn run db:fixtures:load` should snap data back to seed state
* working demonstration of using graphql layer to expose a subset of prisma functionality
* demonstration of jwt parsing, tenant header parsing & usage
